types/ApiResponses.ts

```diff
@@ -452,3 +452,7 @@ export interface PatchResumeResponse extends ApiResponse {
 }
 
 export interface DeleteResumeResponse extends ApiResponse {}
+
+export interface CancelMerchOrderResponse extends ApiResponse {
```

Comment:
Can this (and possibly any other Merch Store related return types be moved to the correct part of this file?

Reply:
Feel free to clean this file up if you want it feels very messy at the bottom